store/dataStore.ts

```diff
@@ -64,48 +65,45 @@ const initialState: IDataStoreState = {
 export const useDataStore = create<DataStore>((set) => ({
   ...initialState,
   resetUser: () => set({ user: initialState.user }),
-  updateNFLTeam: (updatedTeam: INFLTeam): void =>
+  updateNFLTeam: (teams: INFLTeam[]): void =>
```

Comment:
Why was this switched to an array? A user can only update 1 pick with 1 team.

Reply:
This is not for picks.  This is for the array of NFL Teams that is returned from Appwrite.  The user pick does not affect the NFLTeam data store object.